python/tidy/servo_tidy/tidy.py

```diff
@@ -1062,6 +1062,9 @@ def check_commits(path='.'):
         if 'wip' in commit.split():
             yield ('.', 0, 'no commits should contain WIP')
 
+    if len(commits) > 1:
```

Comment:
I think this will fail regardless of whether it's a merge commit. Because, at this point, `commits` is simply all the commits since the last merge.